docusaurus.config.js

```diff
@@ -12,13 +12,20 @@ module.exports = {
   projectName: 'docs', // Usually your repo name.
   trailingSlash: false,
   themeConfig: {
+    prism: {
+      // Enable extra languages when doing syntax highlighting
+      additionalLanguages: [
+        'rust',
+        'rego',
+      ],
+    },
     colorMode: {
       // "light" | "dark"
       defaultMode: "light",
 
       // Hides the switch in the navbar
       // Useful if you want to support a single color mode
-      disableSwitch: true,
+      disableSwitch: false,
```

Comment:
I'd rather we keep this as true since we don't have the theming for dark mode yet.
```suggestion
      disableSwitch: false,
```

Reply:
Everything is working fine an looking natural here. I would enable this option, we can always tune the theme later on if needed (but right now I personally don't see the need)

# Dark mode

![image](https://user-images.githubusercontent.com/22728/172579010-2a4402ba-60a2-44ee-bf1d-e4d9dcf3b5dc.png)

# Light mode

![image](https://user-images.githubusercontent.com/22728/172579104-4247732d-b31a-4d07-9c00-9eba44d8852d.png)

Reply:
I don't have any preference since I was told by the UI/UX team that the theming was only for light mode. If y'all are happy, I am good too!